litho-core/build.gradle

```diff
@@ -70,6 +70,7 @@ dependencies {
 
     // Android Support Library
     compileOnly deps.supportAnnotations
+    implementation deps.guava
```

Comment:
Are there any non-guava alternatives here? Multiset is useful but guava can be a lightning rod dependency. Or is FB Android code ok with Guava?

Reply:
There are other places within Litho that are currently using guava, check litho/lib/guava/BUCK

Reply:
I got the impression it was only test code, n.p. if it's already used then.

Reply:
It's only used in the annotation processor and test code so it doesn't get shipped in any apps using Litho.

Reply:
I see, thanks for the context. I will remove usages of MultiSet from the ExecutorLithoHandler, and also the dependency on guava.